test/mpfr.jl

```diff
@@ -12,7 +12,7 @@ import Base.MPFR
     @test x == BigFloat(x) == BigFloat(0xc) == BigFloat(12.) ==
           BigFloat(BigInt(12)) == BigFloat(BigFloat(12)) == parse(BigFloat,"12") ==
           parse(BigFloat,"12 ") == parse(BigFloat," 12") == parse(BigFloat," 12 ") ==
-          BigFloat(Float32(12.)) == BigFloat(12//1)
+          BigFloat(Float32(12.)) == BigFloat(12//1) == parse(BigFloat, strip(" 12"))
```

Comment:
Change `strip(" 12")` to `SubString("12")` in case the outout type of `strip` changes in the future?

Reply:
Yes, good idea. Updating in a bit.